package.json

```diff
@@ -18,63 +18,45 @@
 	},
 	"dependencies": {
 		"@woocommerce/components": "^12.3.0",
-		"@woocommerce/currency": "^4.3.0",
-		"@woocommerce/customer-effort-score": "^3.0.0",
-		"@woocommerce/data": "^5.0.0",
-		"@woocommerce/date": "^4.3.0",
 		"@woocommerce/navigation": "^8.2.0",
-		"@woocommerce/number": "^2.5.0",
 		"@woocommerce/tracks": "^1.4.0",
-		"@wordpress/api-fetch": "^6.39.13",
-		"@wordpress/base-styles": "^4.33.13",
-		"@wordpress/components": "^25.8.14",
-		"@wordpress/compose": "^6.19.13",
-		"@wordpress/data": "^9.12.13",
-		"@wordpress/dataviews": "^4.17.0",
-		"@wordpress/date": "^4.42.13",
-		"@wordpress/dom": "^3.42.13",
-		"@wordpress/element": "^5.19.13",
-		"@wordpress/hooks": "^3.58.0",
-		"@wordpress/html-entities": "^3.42.13",
-		"@wordpress/i18n": "^4.42.13",
-		"@wordpress/icons": "^9.33.13",
-		"@wordpress/primitives": "^3.40.13",
-		"@wordpress/url": "^3.43.13",
-		"@wordpress/viewport": "^6.22.0",
+		"@wordpress/api-fetch": "^7.26.0",
+		"@wordpress/base-styles": "^6.2.0",
+		"@wordpress/components": "^29.12.0",
+		"@wordpress/compose": "^7.26.0",
+		"@wordpress/data": "^10.26.0",
+		"@wordpress/element": "^6.26.0",
+		"@wordpress/hooks": "^4.26.0",
+		"@wordpress/i18n": "^5.26.0",
+		"@wordpress/url": "^4.26.0",
 		"classnames": "^2.5.1",
 		"gridicons": "^3.4.2",
-		"lodash": "^4.17.21",
-		"prop-types": "^15.8.1",
-		"rememo": "^4.0.2"
+		"lodash": "^4.17.21"
 	},
 	"devDependencies": {
-		"@hapi/h2o2": "^10.0.4",
-		"@hapi/hapi": "^21.3.10",
 		"@playwright/test": "^1.45.3",
 		"@testing-library/dom": "^10.4.0",
-		"@testing-library/jest-dom": "^5.17.0",
+		"@testing-library/jest-dom": "^6.6.3",
 		"@testing-library/react": "^16.0.0",
 		"@testing-library/user-event": "^14.5.2",
-		"@woocommerce/dependency-extraction-webpack-plugin": "^3.0.1",
+		"@woocommerce/dependency-extraction-webpack-plugin": "^4.0.0",
 		"@woocommerce/eslint-plugin": "^2.3.0",
 		"@wordpress/env": "^10.4.0",
-		"@wordpress/jest-preset-default": "^11.29.0",
-		"@wordpress/prettier-config": "^3.15.0",
-		"@wordpress/scripts": "^27.9.0",
+		"@wordpress/jest-preset-default": "^12.26.0",
+		"@wordpress/prettier-config": "^4.26.0",
+		"@wordpress/scripts": "^30.19.0",
```

Comment:
Let's be careful on this one, I feel like bumping this one sometimes impacts things unexpectedly and should be tested on this bump along